README.md

```diff
@@ -429,6 +434,78 @@ override func prepare(for segue: UIStoryboardSegue, sender: Any?) {
 }
 ```
 
+## JSON and YAML
+
+```sh
+swiftgen json -t swift4 /path/to/json/dir-or-file
+swiftgen yaml -t swift4 /path/to/yaml/dir-or-file
+```
+
+This will parse the given file, or when given a directory, recursively search for JSON and YAML files. It will define an `enum` for each file (and documents in a file where needed), and type-safe constants for the content of the file.
```

Comment:
Given that those parsers are a little special in the sense that they are more likely to be used with custom templates than the bundled one, maybe mention something about it there. Something in the likes of:

> Unlike other subcommands, those parsers are intended to allow you to use more custom inputs (as the formats are quite open to your needs) to generate your custom code. This means that for these subcommands, you'll probably be more likely to use custom templates to generate your custom code properly adapted/tuned to your custom YAML/JSON inputs than using the bundled templates. To read more about writing your own custom templates tuned to your custom inputs, see [link to doc]

(well, maybe not that verbose, but you see the gist)